### DISCUSSION TITLE
Stand up Wednesday May 24th

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#83] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on Adan's stuff

**What are your priorities for the next 24 hours?**:
- Continue working on `Más cursadas` graph

**Blockers**:
- None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
-  Made a PR on aux function for dowloanding dataset and forward to analysis service

**What are your priorities for the next 24 hours?**:
- work with Jacob to transfer graph data to main server

**Blockers**:
- None

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Most trendy IBM certs graph
- Most popular courses graph
- Least trendy IBM certs graph?

**What are your priorities for the next 24 hours?**:
- Finish the other graphs
- Turn them into endpoints

**Blockers**:
- None
